Source/Tests/Trakt.NET.Tests/Objects/Get/Collections/Implementations/TraktCollectionMovie_Tests.cs

```diff
@@ -189,39 +193,41 @@ public async Task Test_TraktCollectionMovie_From_Full_Json()
         private const string FULL_JSON =
             @"{
                 ""collected_at"": ""2014-09-01T09:10:11.000Z"",
-                ""movie"": {
-                  ""title"": ""Star Wars: The Force Awakens"",
-                  ""year"": 2015,
-                  ""ids"": {
-                    ""trakt"": 94024,
-                    ""slug"": ""star-wars-the-force-awakens-2015"",
-                    ""imdb"": ""tt2488496"",
-                    ""tmdb"": 140607
-                  },
-                  ""tagline"": ""Every generation has a story."",
-                  ""overview"": ""Thirty years after defeating the Galactic Empire, Han Solo and his allies face a new threat from the evil Kylo Ren and his army of Stormtroopers."",
-                  ""released"": ""2015-12-18"",
-                  ""runtime"": 136,
-                  ""trailer"": ""http://youtube.com/watch?v=uwa7N0ShN2U"",
-                  ""homepage"": ""http://www.starwars.com/films/star-wars-episode-vii"",
-                  ""rating"": 8.31988,
-                  ""votes"": 9338,
-                  ""updated_at"": ""2016-03-31T09:01:59Z"",
-                  ""language"": ""en"",
-                  ""available_translations"": [
-                    ""en"",
-                    ""de"",
-                    ""en"",
-                    ""it""
-                  ],
-                  ""genres"": [
-                    ""action"",
-                    ""adventure"",
-                    ""fantasy"",
-                    ""science-fiction""
-                  ],
-                  ""certification"": ""PG-13"",
-                  ""country"": ""us""
+                ""updated_at"": ""2014-09-01T09:10:11.000Z"",
+                ""movie"": {  
+                   ""title"": ""Star Wars: The Force Awakens"",
+                   ""year"": 2015,
+                   ""ids"": {  
+                      ""trakt"": 94024,
+                      ""slug"": ""star-wars-the-force-awakens-2015"",
+                      ""imdb"": ""tt2488496"",
+                      ""tmdb"": 140607
+                   },
+                   ""tagline"": ""Every generation has a story."",
+                   ""overview"": ""Thirty years after defeating the Galactic Empire, Han Solo and his allies face a new threat from the evil Kylo Ren and his army of Stormtroopers."",
+                   ""released"": ""2015-12-18"",
+                   ""runtime"": 136,
+                   ""country"": ""us"",
```

Comment:
Please revert this line change. The "country" property was previously at the end.